sphinx/ext/inheritance_diagram.py

```diff
@@ -334,6 +345,7 @@ class InheritanceDiagram(SphinxDirective):
     final_argument_whitespace = True
     option_spec = {
         'parts': int,
+        'style': str,
```

Comment:
How about using `docutils.parsers.rst.directive:choice` instead? Then you don't need to validate the value manually.